### PR TITLE
Added Open links to Task Screens and Scripts

### PR DESCRIPTION
**Pull request by lisa-fehr**
Closes https://github.com/ProcessMaker/modeler/issues/942

### CODE REVIEW

File: resources/js/processes/modeler/components/inspector/ScreenSelect.vue [hunk 1]
```diff
@@ -23,6 +23,14 @@
         </multiselect>
         <small v-if="error" class="text-danger">{{ error }}</small>
         <small v-if="helper" class="form-text text-muted">{{ $t(helper) }}</small>
+        <a
+                v-if="content.id"
+                :href="`/designer/screens/${content.id}/edit`"
+                target="_blank"
+        >
+            Open Screen
```
Comment (chipit24) [inline resources/js/processes/modeler/components/inspector/ScreenSelect.vue]: Are translations available for this link text? Could it be wrapped in `$t`?

File: resources/js/processes/modeler/components/inspector/ScreenSelect.vue [hunk 2]
```diff
@@ -23,6 +23,14 @@
         </multiselect>
         <small v-if="error" class="text-danger">{{ error }}</small>
         <small v-if="helper" class="form-text text-muted">{{ $t(helper) }}</small>
+        <a
+                v-if="content.id"
+                :href="`/designer/screens/${content.id}/edit`"
```
Comment (chipit24) [inline resources/js/processes/modeler/components/inspector/ScreenSelect.vue]: Should this go to `/designer/screen-builder/${content.id}/edit` instead? The current URL seems to open a page to edit only the name, description, and category, and not the screen builder.

File: resources/js/processes/modeler/components/inspector/ScreenSelect.vue (hunk 1 above)
Comment (lisa-fehr) [reply resources/js/processes/modeler/components/inspector/ScreenSelect.vue]: No translations. Should I wrap it?

File: resources/js/processes/modeler/components/inspector/ScreenSelect.vue (hunk 2 above)
Comment (lisa-fehr) [reply resources/js/processes/modeler/components/inspector/ScreenSelect.vue]: That is the link I was taken to. Let me check.

File: resources/js/processes/modeler/components/inspector/ScreenSelect.vue (hunk 1 above)
Comment (lisa-fehr) [reply resources/js/processes/modeler/components/inspector/ScreenSelect.vue]: "Open Process" isn't wrapped. So I followed that.

Comment (chipit24) [reply resources/js/processes/modeler/components/inspector/ScreenSelect.vue]: I should have wrapped "Open Process". I think it makes sense to, as everything seems to be translated. Worst case is the translation doesn't exist and it shows the text as-is.

**Pull request merged**